.github/workflows/e2e-certifier.yml

```diff
@@ -0,0 +1,34 @@
+name: 'End-to-end certifier'
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment in which to run the job
+      region:
+        description: The region in which to run the job
+      package:
+        description: The package name
+      job:
+        description: The name of the job (as defined in the deployment config)
+
+jobs:
+  test-job:
+    runs-on: [ "coveo env=${{ inputs.environment }} region=${{ inputs.region }}", arm64, linux, eks ]
+    steps:
+      - name: Deploy JSUI beta version on Netlify
+        run: curl --request POST https://api.netlify.com/build_hooks/${{ secrets.NETLIFY_BUILD_HOOK_ID }}
```

Comment:
In the Playwright tests, we might need to add a project that runs before the main project. It would wait for the `Coveo.version` value on the Netlify page to be the expected version. (It would scrape the NPM repo to know the expected `beta` version.